code/index.html

```diff
@@ -1,22 +1,116 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta cnarset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" href="./img/favicon-2.png" type="image/x-icon">
+  <title>Japan Travel Magazine</title>
+</head>
+
+<body>
+  <header>
+    <div class="container clear">
+      <div class="header-left">
+        <h1 class="header-nav-logo">
+          <a href="#"><img src="./img/logo.png" alt="">
```

Comment:
I think the alt attribute shall contain a brief description of the image. This to be helped if he image wouldnt load properly.